requirements.txt

```diff
@@ -4,4 +4,4 @@ click
 psutil
 rpy2==3.3.6
 nchelpers==5.5.7
-wps_tools==1.0.2
+wps_tools==1.2.0
```

Comment:
Did we miss an update here? Or did the package jump straight from `1.0.2` to `1.2.0`?

Reply:
`1.1.0` just added the `csv_handler` which isn't used in quail